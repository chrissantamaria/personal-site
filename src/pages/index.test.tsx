import React from 'react';
import renderer from 'react-test-renderer';

import IndexPage from '.';

describe('IndexPage', () => {
  it('renders', () => {
    const tree = renderer.create(<IndexPage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
