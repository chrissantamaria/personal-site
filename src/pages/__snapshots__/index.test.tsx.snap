// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IndexPage renders 1`] = `
<div
  className="css-j7qwjs"
>
  <div
    className="css-3ay0rc"
  >
    <h1
      className="css-1hc6z4q"
    >
      Chris Santamaria
    </h1>
    <h2
      className="css-1aqd1sl"
    >
      Student, software engineer, and avid learner
    </h2>
    <a
      css={
        Object {
          "bottom": 8,
          "cursor": "pointer",
          "left": "50%",
          "position": "absolute",
          "transform": "translateX(-50%)",
          "transition": "opacity 0.5s",
        }
      }
      href="#about"
      style={
        Object {
          "opacity": 1,
          "pointerEvents": "auto",
        }
      }
    >
      <svg
        css={
          Object {
            "animation": "_EMO_animation-2uqv96_@keyframes animation-2uqv96{
  0% {
    transform: translateY(0px);
  } 
  50% {
    transform: translateY(-8px);
  } 
  100% {
    transform: translateY(0px);
  }
}_EMO_ 2s ease infinite",
            "fill": "white",
            "height": 48,
          }
        }
        viewBox="0 0 24 24"
      >
        <title>
          Arrow
        </title>
        <path
          d="M12 16a1 1 0 0 1-.64-.23l-6-5a1 1 0 1 1 1.28-1.54L12 13.71l5.36-4.32a1 1 0 0 1 1.41.15 1 1 0 0 1-.14 1.46l-6 4.83A1 1 0 0 1 12 16z"
        />
      </svg>
    </a>
  </div>
  <div
    className="css-1lva9lf"
    id="about"
  >
    <h3
      className="css-1mis095"
    >
      Hi, I'm Chris! ðŸ‘‹
    </h3>
    <p
      className="css-q9kmgt"
    >
      I'm a student at the University of Virginia studying computer science, currently building web experiences at
       
      <a
        className="css-q2y3yl"
        href="https://lyft.com"
        onClick={[Function]}
        rel="noreferrer"
        target="_blank"
      >
        Lyft
      </a>
      .
    </p>
    <p
      className="css-q9kmgt"
    >
      Feel free to reach out through any of the platforms below or view my resume. I'd love to get in touch!
    </p>
    <div
      className="css-p4sd7u"
    >
      <a
        css={
          Object {
            "margin": "0 1rem",
          }
        }
        href="https://github.com/chrissantamaria"
        onClick={[Function]}
        rel="noreferrer"
        target="_blank"
      >
        <svg
          css={
            Object {
              "height": 36,
            }
          }
          viewBox="0 0 24 24"
        >
          <title>
            GitHub
          </title>
          <path
            d="M12 .297c-6.63 0-12 5.373-12 12 0 5.303 3.438 9.8 8.205 11.385.6.113.82-.258.82-.577 0-.285-.01-1.04-.015-2.04-3.338.724-4.042-1.61-4.042-1.61C4.422 18.07 3.633 17.7 3.633 17.7c-1.087-.744.084-.729.084-.729 1.205.084 1.838 1.236 1.838 1.236 1.07 1.835 2.809 1.305 3.495.998.108-.776.417-1.305.76-1.605-2.665-.3-5.466-1.332-5.466-5.93 0-1.31.465-2.38 1.235-3.22-.135-.303-.54-1.523.105-3.176 0 0 1.005-.322 3.3 1.23.96-.267 1.98-.399 3-.405 1.02.006 2.04.138 3 .405 2.28-1.552 3.285-1.23 3.285-1.23.645 1.653.24 2.873.12 3.176.765.84 1.23 1.91 1.23 3.22 0 4.61-2.805 5.625-5.475 5.92.42.36.81 1.096.81 2.22 0 1.606-.015 2.896-.015 3.286 0 .315.21.69.825.57C20.565 22.092 24 17.592 24 12.297c0-6.627-5.373-12-12-12"
          />
        </svg>
      </a>
      <a
        css={
          Object {
            "margin": "0 1rem",
          }
        }
        href="https://www.linkedin.com/in/chris-santamaria-10"
        onClick={[Function]}
        rel="noreferrer"
        target="_blank"
      >
        <svg
          css={
            Object {
              "height": 36,
            }
          }
          viewBox="0 0 24 24"
        >
          <title>
            LinkedIn
          </title>
          <path
            d="M20.447 20.452h-3.554v-5.569c0-1.328-.027-3.037-1.852-3.037-1.853 0-2.136 1.445-2.136 2.939v5.667H9.351V9h3.414v1.561h.046c.477-.9 1.637-1.85 3.37-1.85 3.601 0 4.267 2.37 4.267 5.455v6.286zM5.337 7.433c-1.144 0-2.063-.926-2.063-2.065 0-1.138.92-2.063 2.063-2.063 1.14 0 2.064.925 2.064 2.063 0 1.139-.925 2.065-2.064 2.065zm1.782 13.019H3.555V9h3.564v11.452zM22.225 0H1.771C.792 0 0 .774 0 1.729v20.542C0 23.227.792 24 1.771 24h20.451C23.2 24 24 23.227 24 22.271V1.729C24 .774 23.2 0 22.222 0h.003z"
          />
        </svg>
      </a>
      <a
        css={
          Object {
            "margin": "0 1rem",
          }
        }
        href="/resume.pdf"
        onClick={[Function]}
        rel="noreferrer"
        target="_blank"
      >
        <svg
          css={
            Object {
              "height": 36,
            }
          }
          viewBox="0 0 512 512"
        >
          <title>
            Resume
          </title>
          <path
            d="M428 224H288a48 48 0 01-48-48V36a4 4 0 00-4-4h-92a64 64 0 00-64 64v320a64 64 0 0064 64h224a64 64 0 0064-64V228a4 4 0 00-4-4z"
          />
          <path
            d="M419.22 188.59L275.41 44.78a2 2 0 00-3.41 1.41V176a16 16 0 0016 16h129.81a2 2 0 001.41-3.41z"
          />
        </svg>
      </a>
      <a
        css={
          Object {
            "margin": "0 1rem",
          }
        }
        href="mailto:chris@santamaria.me?subject=Hey!"
        onClick={[Function]}
        rel="noreferrer"
        target="_blank"
      >
        <svg
          css={
            Object {
              "height": 36,
            }
          }
          viewBox="0 0 24 24"
        >
          <title>
            Email
          </title>
          <path
            d="M20 4H4c-1.1 0-1.99.9-1.99 2L2 18c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V6c0-1.1-.9-2-2-2zm0 4l-8 5-8-5V6l8 5 8-5v2z"
          />
        </svg>
      </a>
    </div>
  </div>
</div>
`;
